MFaaP.MFWSClient.Tests/MFWSVaultValueListItemOperations.cs

```diff
@@ -149,6 +149,41 @@ public void GetValueListItems_WithLimitAndNameFilter()
 			runner.Verify();
 		}
 
+		/// <summary>
+		/// Ensures that a call to <see cref="MFaaP.MFWSClient.MFWSVaultValueListItemOperations.CreateValueListItemAsync"/>
+		/// requests the correct resource using the correct method.
+		/// </summary>
+		[TestMethod]
+		public async Task  CreateValueListItemAsync()
+        {
+			// Create our test runner.
+			var runner = new RestApiTestRunner<ValueListItem>(Method.POST, "/REST/valuelists/1/items");
+
```

Comment:
Please alter the test to also validate the expected POST body (see [this](https://github.com/M-Files/Libraries.MFWSClient/blob/master/MFaaP.MFWSClient.Tests/MFWSVaultObjectOperations.cs#L31) as an example).